BenFradet/RiotSharp
# [Develop] Added support for current-game-v1.0 and featured-games-v1.0 endpoints [fixes #107]

Support for the two newest endpoints to the API added.

Manually tested basic functionality:
+ GetCurrentGame on self in ARAM game: successful response and correct content
+ GetCurrentGame on self in custom game: 404 error (waiting on a reply from devs on the api discussion to see if customs are available through the endpoint or not)
+ GetFeaturedGames on NA region: successful response and matched content that was displayed in the client at the time

I'll try to get some unit tests done in the next few days when I have some more time.
